teku/src/main/java/tech/pegasys/teku/cli/converter/GraffitiConverter.java

```diff
@@ -16,22 +16,21 @@
 import java.nio.charset.StandardCharsets;
 import org.apache.tuweni.bytes.Bytes32;
 import picocli.CommandLine;
+import tech.pegasys.teku.validator.client.loader.graffiti.Bytes32Converter;
 
 public class GraffitiConverter implements CommandLine.ITypeConverter<Bytes32> {
   @Override
   public Bytes32 convert(final String value) {
     byte[] input = value.getBytes(StandardCharsets.UTF_8);
-    if (input.length > 32) {
+    try {
+      return Bytes32Converter.toBytes32(input);
```

Comment:
(nit) May be worth adding a method to  `Bytes32Converter` that directly accepts a `String` as well.